client/pod.go

```diff
@@ -54,7 +54,7 @@ func (cli *HyperClient) CreatePod(jsonbody string) (string, error) {
 	v := url.Values{}
 	v.Set("podArgs", jsonbody)
 	body, statusCode, err := readBody(cli.call("POST", "/pod/create?"+v.Encode(), nil, nil))
-	if statusCode == 404 && strings.Contains(err.Error(), "could not find image: no such id") {
+	if statusCode == 404 || (err != nil && strings.Contains(err.Error(), "could not find image: no such id")) {
```

Comment:
What's the return value of `readBody` in case the image does not exist? What's the difference between the 2 cases?